src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/output_format_converter.py

```diff
@@ -0,0 +1,275 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from typing import Dict, List
+
+from genai_perf.exceptions import GenAIPerfException
+from genai_perf.llm_inputs.shared import (
+    DEFAULT_TENSORRTLLM_MAX_TOKENS,
+    ModelSelectionStrategy,
+    OutputFormat,
+)
+
+
+class OutputFormatConverterFactory:
```

Comment:
Should we change the name "OutputFormat" into something else with this refactor? I feel like "output" is a bit misleading term. It's definitely output format for llminputs but more precisely, this is specifying the format of request inputs. How about
- `RequestFormat`
- `InputFormat`
- `PayloadFormat`
- etc.

I prefer `RequestFormat` since data parser has something similar called `ResponseFormat` for parsing the responses.

Reply:
I think this violates the idea that when we do refactors, we try not to change other stuff.
I think we risk losing this change in the noise of the files moving.
I would prefer this is a separate PR but I agree it needs to be updated.
I like `RequestFormat` as well.

Reply:
That's a fair point. @dyastremsky If this seems to require too much change, we can consider this out of scope for this PR.

Reply:
Handling this in a future PR.